src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/srm/matcher/SymbolicRegexMatcher.cs

```diff
@@ -484,6 +496,13 @@ private State<S> CreateNewTransition(State<S> q, S atom, int offset)
             }
         }
 
+        [MethodImpl(MethodImplOptions.AggressiveInlining)]
```

Comment:
It's not important right now, but FYI we don't put AggressiveInlining on small methods like this as they will almost surely be inlined.

Reply:
good to know. Will remove those during a later cleanup.